src/components/molecules/Visualizer/Infobox/Frame/index.tsx

```diff
@@ -222,6 +236,10 @@ const Content = styled.div<{ open?: boolean }>`
 
   max-height: ${({ open }) => (open ? "50vh" : "0")};
   padding: ${({ open }) => (open ? "20px 0" : "0")};
+  padding-top: ${({ paddingTop }) => paddingTop && `${paddingTop}px`};
+  padding-bottom: ${({ paddingBottom }) => paddingBottom && `${paddingBottom}px`};
+  padding-left: ${({ paddingLeft }) => paddingLeft && `${paddingLeft}px`};
+  padding-right: ${({ paddingRight }) => paddingRight && `${paddingRight}px`};
```

Comment:
```suggestion
  padding-top: ${({ paddingTop }) => paddingTop ?`${paddingTop}px` : "0"};
  padding-bottom: ${({ paddingBottom }) => paddingBottom ? `${paddingBottom}px` : "0"};
  padding-left: ${({ paddingLeft }) => paddingLeft ? `${paddingLeft}px` : "0"};
  padding-right: ${({ paddingRight }) => paddingRight ? `${paddingRight}px` : "0"};
```

Reply:
but this way we will override the default padding
`padding: ${({ open }) => (open ? "20px 0" : "0")};`